doc/articles/uno-community-toolkit.md

```diff
@@ -143,31 +143,32 @@ Type universe cannot resolve assembly: Uno.UI, Version=255.255.255.255,
 Culture=neutral, PublicKeyToken=null.
 ```
 
-To fix this, instead of adding the Uno version of the toolkit like the code below:
+To fix this, instead of adding the Uno version of the toolkit like the code below.
 
 # [WinUI / WinAppSDK](#tab/tabid-winui)
 
+Replace:
 ```xml
 <ItemGroup>
   <PackageReference Include="Uno.CommunityToolkit.WinUI.UI.Controls" Version="7.1.100" />
 </ItemGroup>
 ```
 
-Add a conditional reference:
+With a conditional reference:
 
 ```xml
-<ItemGroup Condition="'$(TargetFramework)' == 'net6.0-windows10.0.18362.0'">
+<ItemGroup Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'windows'">
```

Comment:
This will apply to the WPF build too, is it intended?

Reply:
In a class library, there's no `net7.0-windows` without the additional version, so yes, this is fine.

Reply:
Ah yes. Sorry, forgot that `net7.0-windows` is only in the Wpf-specific project but not in the classlib.